### PR TITLE
Kickoff api

### PR DESCRIPTION
**Pull request by micaelacole**
# Summary
Creation of WBooks app and configuration of the associated PostgreSQL database.

# Trello Card
https://trello.com/c/yd3zQInS

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+# if using bower-rails ignore default bower_components path bower.json files
+/vendor/assets/bower_components
+*.bowerrc
+bower.json
+
+# Ignore pow environment settings
+#.powenv
+
+# Ignore Byebug command history file.
+#.byebug_history
```
Comment (nicozar95) [inline .gitignore]: We dont need the `.byebug_history` in our repo, descoment this

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+# Ignore node_modules
+# node_modules/
```
Comment (nicozar95) [inline .gitignore]: We dont need the `node_moudes`in our repo, descoment this

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+# unless supporting rvm < 1.11.0 or doing something fancy, ignore this:
+.rvmrc
+
+# if using bower-rails ignore default bower_components path bower.json files
+/vendor/assets/bower_components
+*.bowerrc
+bower.json
+
+# Ignore pow environment settings
+#.powenv
```
Comment (nicozar95) [inline .gitignore]: We dont need the `.powenv` in our repo, descoment this

**Review by nicozar95 — CHANGES_REQUESTED** [review]
Great work!! just a couple of changes in the `.gitignore` 😄

File: config/database.yml [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+#   DATABASE_URL="postgres://myuser:mypass@localhost/somedatabase"
+#
+# You can use this database configuration with:
+#
+#   production:
+#     url: <%= ENV['DATABASE_URL'] %>
+#
+production:
+  <<: *default
+  database: wbooks-api_production
```
Comment (facucompe) [inline config/database.yml]: `url: <%= ENV['DATABASE_URL'] %>` we use URL for production

File: config/database.yml [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+#
+default: &default
+  adapter: postgresql
+  encoding: unicode
+  # For details on connection pooling, see Rails configuration guide
+  # http://guides.rubyonrails.org/configuring.html#database-pooling
+  host: localhost
+  port: 5432
+  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  username: 'wbooks-api'
```
Comment (facucompe) [inline config/database.yml]: Remember this fields should search an env variable: `username: <%= ENV.fetch('DB_USERNAME', 'wbooks-api') %>`

Comment (facucompe) [reply config/database.yml]: Same for: host, port, and password

**Review by facucompe — CHANGES_REQUESTED** [review]

File: config/database.yml (hunk 5 above)
Comment (micaelacole) [reply config/database.yml]: Okay, I'll change all these lines, but should I leave the specification of ENV variables for later? I thought I was told I would be requested to do this task (defining ENV variables and .env file) later in another card, but I'm not so sure now.

File: config/database.yml (hunk 4 above)
Comment (micaelacole) [reply config/database.yml]: Is the url used _instead of_ the database parameter or should I leave both lines?

File: config/database.yml (hunk 5 above)
Comment (nicozar95) [reply config/database.yml]: Add it here, in a future task you will have to manage sensible data of our application and you would exploded better the ENV potential. But, as you used ENV before in this PR, change the database config to use ENV var

File: config/database.yml (hunk 4 above)
Comment (facucompe) [reply config/database.yml]: yes, the url contains also the database name. You should remove the `database` field

**Review by facucompe — CHANGES_REQUESTED** [review]

File: config/database.yml [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+# You can use this database configuration with:
+#
+#   production:
+#     url: <%= ENV['DATABASE_URL'] %>
+#
+production:
+  <<: *default
+  database: wbooks-api_production
+  url: <%= ENV['DATABASE_URL'] %>
+ # password: <%= ENV['WBOOKS_DATABASE_PASSWORD'] %> # era el original
```
Comment (nicozar95) [inline config/database.yml]: Delete this comment, also it's in spanish

**Review by nicozar95 — CHANGES_REQUESTED** [review]
Small change

**Review by nicozar95 — APPROVED** [review]
👍 Great work!

**Review by facucompe — APPROVED** [review]

**Pull request merged**